docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -77,3 +84,33 @@ source /opt/intel/oneapi/setvars.sh
 export USE_XETLA=OFF
 export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
 ```
+
+## Installing from Wheels
+
+You can also install BigDL-LLM from source achieves. First you need to install the target torch/torchvision/ipex versions from download whls [here](http://ec2-52-27-27-201.us-west-2.compute.amazonaws.com/ipex-release.php?device=xpu&repo=us&release=stable) before installing bigdl-llm. 
```

Comment:
install BigDL-LLM -> install BigDL-LLM dependencies for Intel XPU
download -> downloaded